stubs/docker/docker/models/containers.pyi

```diff
@@ -57,7 +57,7 @@ class Container(Model):
     def top(self, **kwargs): ...
     def unpause(self): ...
     def update(self, **kwargs): ...
-    def wait(self, **kwargs): ...
+    def wait(self, *, timeout: int = ..., condition: Literal["not-running", "next-exit", "removed"] = "not-running"): ...
```

Comment:
The timeout can be `None`, which will presumably just wait forever instead of timing out eventually:

```suggestion
    def wait(self, *, timeout: int | None = None, condition: Literal["not-running", "next-exit", "removed"] = "not-running"): ...
```

Technically the condition can also be `None`, but that's kind of useless since it is basically same as `"not-running"`.

Reply:
I think it also accepts floats for the timeout. The timeout gets passed to `requests.Session.post()`.

Reply:
Thank you @Akuli , I have made the suggested change, and the potentially useless change.